src/test/contentMetaConfig.js

```diff
@@ -0,0 +1,27 @@
+const whiteListedChannelList = process.env.sunbird_content_service_whitelisted_channels
+const blackListedChannelList = process.env.sunbird_content_service_blacklisted_channels
+const whitelistedFrameworkList = process.env.sunbird_content_service_whitelisted_framework
+const blacklistedFrameworkList = process.env.sunbird_content_service_blacklisted_framework
+const whitelistedMimeTypeList = process.env.sunbird_content_service_whitelisted_mimetype
+const blacklistedMimeTypeList = process.env.sunbird_content_service_blacklisted_mimetype
+const whitelistedContentTypeList = process.env.sunbird_content_service_whitelisted_contenttype
+const blacklistedContentTypeList = process.env.sunbird_content_service_blacklisted_contenttype
+const whitelistedResourceTypeList = process.env.sunbird_content_service_whitelisted_resourcetype
+const blacklistedResourceTypeList = process.env.sunbird_content_service_blacklisted_resourcetype
+
+module.exports = Object.freeze({
```

Comment:
why is the object name Freeze?

Reply:
its a js function

Reply:
ok, the freeze function is used I got it, but why only doing this manipulate the string and use it directly.
you are creating the var again in the spec which does not solve the purpose of creating this file please revisit this code